parsec/backend/organization.py

```diff
@@ -371,3 +397,10 @@ async def set_expiration_date(
             OrganizationNotFoundError
         """
         raise NotImplementedError()
+
+    async def update(self, id: OrganizationID, **fields: dict):
```

Comment:
I like the sentinel approach (option 1) better because it's quite explicit on the function side and the caller doesn't have to care about it.

I don't know the context, but it might also be simpler to simply expose two methods:
- `update_expiration_date(self, date: Optional[DateTime])` 
- `update_is_outsider_profile_allowed(self, is_allowed: bool)`

Reply:
> but it might also be simpler to simply expose two methods

This is not doable here because we want to be able to update multiple things with a single method call (given this will turn into a single PostgreSQL transaction, and we don't want to expose the transaction mechanism at the abstract component level)